contrib/snyk/check-for-cve.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+PWD=/Users/user/.../hypershift/contrib/snyk
+
+COMPONENT_TO_CHECK=$1
```

Comment:
Given that the component can only be HO or CPO, my pref would be to simply have 2 scripts that call the same code if necessary.